### DISCUSSION TITLE
Generate GraphQL Type definition without Resolver Map to allow incremental adoption?

### DISCUSSION THREAD

**Discussion by KeithGillette** [MichalLytek/type-graphql#1103] [Q&A] [answered]
In my testing, the `buildSchema` and `buildTypeDefsAndResolvers` functions require the `resolvers` array to not be empty. I'm curious if there's a way to only specify `orphanedTypes` to allow the GraphQL Type definitions to be built by `type-graphql` but the Resolvers map to be specified separately, which would allow us to incrementally migrate an existing TypeScript + GraphQL project to `type-graphql` without having to refactor the entire project at once.

**Comment by MichalLytek** [comment, marked as answer]
See #339
